src/settings/MappingProfiles/ViewMappingProfile.js

```diff
@@ -190,17 +171,23 @@ export class ViewMappingProfile extends Component {
     const tagsEntityLink = `data-import-profiles/mappingProfiles/${mappingProfile.id}`;
     const associations = [...[], ...mappingProfile.parentProfiles, ...mappingProfile.childProfiles];
 
-    return (
-      <Pane
+    const header = renderProps => (
```

Comment:
Move this function to the class methods and rename it, please

Reply:
Done!